code/game/machinery/requests_console.dm

```diff
@@ -282,6 +282,8 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 			if(!istype(M,/mob/new_player) && M.client)
 				to_chat(M, "<b><font size = 3><font color = red>[department] announcement:</font color> [message]</font size></b>")
 				M << sound(announceSound)
+				log_say("[key_name(usr)] (@[usr.x],[usr.y],[usr.z]) has made an announcement from \the [src]: [message]")
```

Comment:
This will produce a message for every mob in the playerlist - one of the downsides of testing on your own is that it wouldn't look weird to get 1 message. Move it out of the loop.